lib/structures/NewsChannel.js

```diff
@@ -34,6 +34,15 @@ class NewsChannel extends TextChannel {
     crosspostMessage(messageID) {
         return this.client.crosspostMessage.call(this.client, this.id, messageID);
     }
+
+    /**
+     * Follow a NewsChannel
+     * @param {String} targetChannelID The ID of the target channel
```

Comment:
The permissions needed to run this should be listed here IMO.

Reply:
Not consistent with other docs.

Reply:
gggg